client/app/components/PlayerHeadToHead.tsx

```diff
@@ -0,0 +1,208 @@
+          fontSize={10}
+        >
+          {props.player1TeamCityShort} • {props.position.toUpperCase()}
+        </Text>
+      </View>
+
+{/**---------------------------------------------------------------- */}
+
+      <Text
+        color={colors.text}
+        textAlign='center'
+        fontFamily={typography.fonts.poppins.semiBold}
+        fontSize={17}
+        marginHorizontal={8}
+
+        position="absolute" 
+        // Must use right instead of left margin
+        // Because want this score's right hand side to always be 72.5 pixels from the center
+        right={WINDOW_WIDTH / 2 - 22.5 + 50} 
+
+      >
+        {props.player1Points}
+      </Text>
+
+{/**---------------------------------------------------------------- */}
+
+      <Box
+        bg={positionBgColor}
+        h={45}
+        w={45}
+        borderRadius={10}
+
+        position="absolute" // Set the position component's position to absolute
+        left={WINDOW_WIDTH / 2 - 22.5} // Calculate the left margin to center the position component
+        right={WINDOW_WIDTH / 2 - 22.5} // Calculate the right margin to center the position component    
+
+        display="flex"
+        justifyContent="center" 
+        alignItems="center" 
+      >
+        <Text
+          color={colors.textBlack}
+          textAlign='center'
+          fontFamily={typography.fonts.poppins.semiBold}
+          fontSize={20}
+        >
+          {props.position.toUpperCase()}
+        </Text>
+      </Box>
+
+{/**---------------------------------------------------------------- */}
+
+      <Text
+        color={colors.text}
+        textAlign='center'
+        fontFamily={typography.fonts.poppins.semiBold}
+        fontSize={17}
+        marginHorizontal={8}
+
+        position="absolute" 
+        // Must use left instead of right margin
+        // Because want this score's left hand side to always be 72.5 pixels from the center
+        left={WINDOW_WIDTH / 2 - 22.5 + 50} 
+
+      >
+        {props.player2Points}
+      </Text>
+
+{/**---------------------------------------------------------------- */}
+
+      <View
+        flexDirection={"column"}
+        alignItems={"flex-end"}
+
+        position="absolute" 
+        // Must use right instead of left margin
+        // Because want component's RHS to alwasy line up
+        right={8} 
+
+      >
+        <Text
+          color={colors.text}
+          textAlign='center'
+          fontFamily={typography.fonts.poppins.semiBold}
+          fontSize={14}
+        >
+          {props.player2NameShort}
+        </Text>
+        <Text
+          color={colors.text}
+          textAlign='center'
+          fontFamily={typography.fonts.poppins.normal}
+          fontSize={10}
+        >
+          {props.player2TeamCityShort} • {props.position.toUpperCase()}
+        </Text>
+      </View>
+
+{/**---------------------------------------------------------------- */}
+
+    </View>
+    
+  );
+};
```

Comment:
Overall great code, I really like all of the left and right alignments in order to give the correct positioning for the various elements